src/components/User/User.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const User = ({ name }) => (
+  <span>{name}</span>
+);
```

Comment:
It's very irrational to create element with only one 'span' that only displays the name of the user. You should put this span in your 'Todo' component

Reply:
made it